python/xorq/cli.py

```diff
@@ -57,6 +57,36 @@ def build_command(script_path, expression, builds_dir="builds"):
     )
 
 
+def run_command(builds_dir, hash_id, output_format="csv"):
+    """
+    Run a build, by recreating the expression from the build
+
+    Parameters
+    ----------
+    builds_dir : Path to the builds directory
+    hash_id : Hash identifier of the build to run
+    output_format : Output format of the run (either "csv" or "json")
+
+    Returns
+    -------
+
+    """
+    try:
+        build_manager = BuildManager(builds_dir)
+        expr = build_manager.load_expr(hash_id)
+        frame = expr.execute()
+
+        match output_format:
```

Comment:
if we `execute`, then we have to pull all the data into memory at once

i think the default format should be `parquet`

Reply:
if the user does not explicitly provide an output format, perhaps the "default" should be writing parquet to `/dev/null`

Reply:
(but still parquet so there's less conversion overhead)

Reply:
If we are going to write files, I prefer to make it so that each format writes to disk